src/main/java/com/arcbees/bourseje/client/application/confirmvote/ConfirmVotePresenter.java

```diff
@@ -74,14 +77,16 @@
     public void prepareFromRequest(PlaceRequest request) {
         super.prepareFromRequest(request);
 
-        name = request.getParameter(NameTokens.VALUE_PARAM, "noSelection");
+        name = request.getParameter(NameTokens.NAME, "noSelection");
+        company = request.getParameter(NameTokens.COMPANY, "noSelection");
 
         getView().setName(name);
+        getView().setCompany(company);
     }
 
     @Override
     public void onConfirmClicked() {
-        VoteItem voteItem = new VoteItem(name);
+        VoteItem voteItem = new VoteItem(name + " " + company);
```

Comment:
Add a TODO here to split "value" into "name" and "company", I'll take care of it. Thanks! :)